cypress/e2e/withMappings/user-journey.cy.ts

```diff
@@ -36,18 +35,25 @@ describe("User journey", () => {
       .should("contain", "Reserve bog");
   });
 
-  // TODO: Fix the failing test. It is not able, for some reason, to press the submit button.
-  it.skip("Can open reservation modal & reserve a material", () => {
+  it("Can open reservation modal & reserve a material", () => {
     const authorizationCode = "7c5e3213aea6ef42ec97dfeaa6f5b1d454d856dc";
     const accessToken = "447131b0a03fe0421204c54e5c21a60d70030fd1";
     const userGuid = "19a4ae39-be07-4db9-a8b7-8bbb29f03da6";
     cy.adgangsplatformenLogin(authorizationCode, accessToken, userGuid);
     cy.visit("/work/work-of:870970-basis:54181744");
     cy.getBySel("material-header-author-text").scrollIntoView();
     cy.getBySel("material-header-buttons-physical").click();
-    cy.getBySel("modal")
+    // We have to wait for the modal to be fully rendered.
+    // eslint-disable-next-line cypress/no-unnecessary-waiting
+    cy.wait(2000);
+    cy.getBySel("reservation-modal-parallel")
       .should("be.visible")
       .and("contain", "Harry Potter og Fønixordenen");
+    // We have to wait for the modal to be fully rendered
+    // and the event listeners to be attached.
+    // Read more: https://www.cypress.io/blog/2019/01/22/when-can-the-test-click/
```

Comment:
Very nice that you added this link! Should we consider using .pipe() like they do? I know it's an extra package, but we could just add it to devDependencies maybe?

Reply:
That could be an idea yes. I think I'll add a todo about for now, so we don't use more time on it.